first_django_pr/my_app/forms.py

```diff
@@ -0,0 +1,57 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from .models import Note
+
+class NoteForm(forms.ModelForm):
+    title = forms.CharField(min_length=5, max_length=10)
+
+    class Meta:
+        model = Note
+        fields = ['msg', 'assignee', 'email']
+
+    def clean_assignee(self):
+        assignee = self.cleaned_data.get('assignee')
+        if assignee and len(assignee.split()) < 2:
```

Comment:
There are one more requirement all words should be started from the capital letter

Reply:
Ok

Reply:
def clean_assignee(self):
        Assignee = self.cleaned_data.get('assignee')
        if Assignee and len(Assignee.split()) < 2:
            raise forms.ValidationError("Assignee should contain at least two words.")
        return Assignee